doc/ref/configuration/minion.rst

```diff
@@ -1725,9 +1725,15 @@ enabled and can be disabled by changing this value to ``False``.
     If ``extmod_whitelist`` is specified, modules which are not whitelisted will also be cleaned here.
 
 .. conf_minion:: environment
+.. conf_minion:: saltenv
 
-``environment``
----------------
+``saltenv``
+-----------
+
+.. versionchanged:: Oxygen
+    Renamed from ``environment`` to ``saltenv``. If ``environment`` is used,
+    ``saltenv`` will take its value. If both are used, ``environment`` will be
+    ignored and ``saltenv`` will be used.
```

Comment:
I am wondering on the effect here for custom modules. Those may need to be updated potentially, yes?

Reply:
The environment config option still exists in opts correct?

Reply:
Correct. If `environment` is used in the config, then `saltenv` and `environment` assume the same value, for just this reason. But, if someone were to reference `__opts__['environment']` in a custom module and `environment` was not defined in the config file, it would no be in the opts dict and there would be a `KeyError`.

Would it make sense to always have both `saltenv` and `environment` both exist, even if neither `saltenv` nor `environment` was defined in the config file?